install.sh

```diff
@@ -23,13 +24,15 @@ esac
 
 deno_uri="https://github.com/denoland/deno/releases/download/${version}/deno_${os}_x64.gz"
 
-curl -fL# -o "$deno_dir/deno.gz" "$deno_uri"
-gunzip -df "$deno_dir/deno.gz"
-chmod +x "$deno_dir/deno"
+curl -fL# -o "$exe.gz" "$deno_uri"
+gunzip -df "$exe.gz"
+chmod +x "$exe"
 
-echo "Deno was installed successfully."
+echo "Deno was installed successfully to $exe"
 if command -v deno >/dev/null; then
   echo "Run 'deno --help' to get started."
 else
-  echo "Run '~/.deno/bin/deno --help' to get started."
+  echo "You must manually add it to your path:"
+  echo "  export PATH=\"$deno_dir:\$PATH\""
```

Comment:
Obviously, this won't persist the PATH variable. But should that be mentioned, too?

I specifically didn't mention how to add deno to the PATH nor how to persist it since I think the target audience is familiar with this.

Reply:
Fixed